test_util/marathon.py

```diff
@@ -290,7 +290,10 @@ def _wait_for_pod_deployment(pod_id):
             r = self.get('v2/pods' + pod_id)
             r.raise_for_status()
             data = r.json()
-            if int(data['scaling']['instances']) != pod_definition['scaling']['instances']:
+            log.debug(data)
+            if 'scaling' in data.keys() and\
+               'scaling' in pod_definition.keys() and\
```

Comment:
This could be much prettier with parenthesis:
```
if ('scaling in data.keys() and 'scaling' in pod_definition.keys' and
        int(data['scaling']['instances']) != pod_definition['scaling']['instances']):
    log.info(...)
    return False
```
If you want each logical condition on its own line, that's fine too, but still use the parenthesis over the line-escapes